frontend/src/metabase/services.js

```diff
@@ -408,6 +408,7 @@ export const UtilApi = {
   random_token: GET("/api/util/random_token"),
   logs: GET("/api/util/logs"),
   bug_report_details: GET("/api/util/bug_report_details"),
+  connection_pool_details_url: "/api/util/diagnostic_info/connection_pool_info",
```

Comment:
I'm not enough of a frontender to tell you what you _should_ do, but it's a little weird to me to have everything else be functions that perform HTTP requests and this just be a string.

Reply:
Yeah, it is a bit weird, but we don't need a verb for this one (since it's a download initiated by the browser visiting a link).